app/routes/joblistings/components/JoblistingList.js

```diff
@@ -6,6 +6,8 @@ import { Image } from 'app/components/Image';
 import Time from 'app/components/Time';
 import { Flex } from 'app/components/Layout';
 import { Year, jobType, Workplaces } from './Items';
+import Tag from 'app/components/Tags/Tag';
+import moment from 'moment';
```

Comment:
I'm quite sure we don't have `moment`, but rather `moment-timezone` 😄 
This should in theory don't work 😕 

`import moment from 'moment-timezone';`

Reply:
fixed it 😎😎😎😎😎